DOC-3034 Focused OSS cluster config article on the config and redirec… (#2981)
RedisLabs/redislabs-docs: content/rs/databases/configure/oss-cluster-api.md

    /rs/databases/configure/oss-cluster-api/,
]
---

> Review the [OSS Cluster API Architecture]({{<relref "/rs/clusters/optimize/oss-cluster-api">}}) to determine if this feature should be enabled for your database.

Reviewer: ```suggestion
Review [Redis OSS Cluster API Architecture]({{<relref "/rs/clusters/optimize/oss-cluster-api">}}) to determine if you should enable this feature for your database.
```
